cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -88,30 +89,6 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 	}
 	t.Logf("Running command: %v", cmd.Args)
 	require.NoError(t, cmd.Start())
-
-	// Wait for the binary to start and become ready to serve requests.
-	healthCheckEndpoint := s.HealthCheckEndpoint
-	if healthCheckEndpoint == "" {
-		healthCheckEndpoint = fmt.Sprintf("http://localhost:%d/", ports.QueryHTTP)
-	}
-	require.Eventually(t, func() bool {
-		t.Logf("Checking if Jaeger-v2 is available on %s", healthCheckEndpoint)
-		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
-		defer cancel()
-		req, err := http.NewRequestWithContext(ctx, http.MethodGet, healthCheckEndpoint, nil)
-		if err != nil {
-			t.Logf("HTTP request creation failed: %v", err)
-			return false
-		}
-		resp, err := http.DefaultClient.Do(req)
-		if err != nil {
-			t.Logf("HTTP request failed: %v", err)
-			return false
-		}
-		defer resp.Body.Close()
-		return resp.StatusCode == http.StatusOK
-	}, 60*time.Second, 3*time.Second, "Jaeger-v2 did not start")
-	t.Log("Jaeger-v2 is ready")
```

Comment:
I moved the healthcheck **after** the call to Cleanup, that was the reason why the binary's logs were never printed (`require` it the HC would abort the test).

Reply:
Ohh okay.